curriculum/challenges/english/01-responsive-web-design/basic-html-and-html5/add-placeholder-text-to-a-text-field.english.md

```diff
@@ -71,7 +71,31 @@ tests:
 ## Solution
 <section id='solution'>
 
-```js
-// solution required
+<div id='html-seed'>
+
+```html
+<h2>CatPhotoApp</h2>
+<main>
+  <p>Click here to view more <a href="#">cat photos</a>.</p>
+
+  <a href="#"><img src="https://bit.ly/fcc-relaxing-cat" alt="A cute orange cat lying on its back."></a>
+
+  <p>Things cats love:</p>
+  <ul>
+    <li>cat nip</li>
```

Comment:
You should capitalize the first letter of the first word of each list item.

Reply:
@itaisteinherz It is just a list, so it is not necessary to capitalize.